src/main/java/com/code_intelligence/jazzer/junit/FuzzTest.java

```diff
@@ -103,9 +103,22 @@
    * A duration string such as "1h 2m 30s" indicating for how long the fuzz test should be executed
    * during fuzzing.
    *
+   * <p>To remove the default limit of 5 minutes, set this element to {@code ""}.
+   *
    * <p>This option has no effect during regression testing.
    */
   String maxDuration() default "5m";
+
+  /**
+   * If set to a positive number, the fuzz test function will be executed at most this many times
+   * during fuzzing.
+   *
+   * <p>Prefer this element over {@link #maxDuration()} if you want to ensure comparable levels of
+   * fuzzing across machine's with different performance characteristics.
+   *
+   * <p>This option has no effect during regression testing.
+   */
+  long maxExecutions() default 0;
```

Comment:
We normally set `-runs=0` to only execute corpus entries. Should the annotation have  a different semantic and couldn't this  be confusing? I'm fine either way, though, but would add `0` as infinite runs to the description.

Reply:
We do have the regression test mode for this, so I don't think `-runs=0` has much use with JUnit-based fuzz tests. But it should still be supported (e.g. for use in cifuzz) as command-line arguments take precedence over this value.

I added an explanation of the default case to the docs.